plugins/common/cookie/cookie.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/influxdata/telegraf/config"
+	"github.com/sirupsen/logrus"
```

Comment:
Please don't do this. We blacklisted `log` as we want a common logging infrastructure to be used. Circumventing the blacklist by using `logrus` is not very nice. ;-)